src/status_im/i18n.cljs

```diff
@@ -260,8 +260,7 @@
   ([path] (label path {}))
   ([path options]
    (if (exists? rn-dependencies/i18n.t)
-     (let [options (update options :amount label-number)]
```

Comment:
What specifically was broken in `label-number`? I'm assuming there was a good reason we had this function, we don't want another bug to pop out elsewhere after this fix. 

Does it make sense to fix `label-number` instead?